contracts/Response.sol

```diff
@@ -154,12 +181,18 @@ contract Response {
   function createQuestion(
     uint64 twitterUserId, bytes32 content, address foundation, uint amount
   ) payable external {
+    // read uint backend fee and make sure it is same as msg.value
     require(msg.value == s.rUint(BACKEND_FEE));
+    // transfer from sender to current address token amount
     require(token.transferFrom(msg.sender, this, amount));
+    // read boolean of foundation addressto check whether it is registered
     require(s.rBool(FOUNDATION, foundation));
     // todo twitter user id and content is unchecked
+    // QUESTION: what will it be checked for, same as foundation boolean?
 
+    // send backend address the backend fee
     s.rAddress(BACKEND).transfer(s.rUint(BACKEND_FEE));
+    // QUESTION: would gas be saved by using questionCount();
```

Comment:
You mean `backendFee` method of Response contract? It does the same, so I think we will lose some gas on calling. Will be cheaper to call `getUint` of Store contract directly, but it is less human-readable.